edx_exams/apps/api/v1/tests/test_views.py

```diff
@@ -1051,29 +1051,22 @@ class ExamAttemptViewTest(ExamsAPITestCase):
     def setUp(self):
         super().setUp()
 
-        self.course_id = 'course-v1:edx+test+f19'
-        self.content_id = '11111111'
-
-        self.course_exam_config = CourseExamConfiguration.objects.create(
-            course_id=self.course_id,
+        self.exam = ExamFactory(
```

Comment:
Just realized, would it make sense for us to make a quick stewardship ticket to replace the exam/exam attempt setups in our test files to these new factories, for consistency's sake?

Reply:
I'm recommending we just refactor these as we go since doing it all at once is a decent chuck of work that doesn't deliver anything new.